common/app/model/liveblog/BlockElement.scala

```diff
@@ -1,24 +1,52 @@
 package model.liveblog
 
 import com.gu.contentapi.client.model.v1.ElementType.{Map => _, _}
-import com.gu.contentapi.client.model.v1.{SponsorshipType, BlockElement => ApiBlockElement, Sponsorship => ApiSponsorship}
+import com.gu.contentapi.client.model.v1.{ElementType, SponsorshipType, BlockElement => ApiBlockElement, Sponsorship => ApiSponsorship}
 import model.{AudioAsset, ImageAsset, ImageMedia, VideoAsset}
 import play.api.libs.json._
 
 sealed trait BlockElement
 case class TextBlockElement(html: Option[String]) extends BlockElement
+case class TweetBlockElement(html: Option[String]) extends BlockElement
+case class PullquoteBlockElement(html: Option[String]) extends BlockElement
 case class ImageBlockElement(media: ImageMedia, data: Map[String, String], displayCredit: Option[Boolean]) extends BlockElement
 case class AudioBlockElement(assets: Seq[AudioAsset]) extends BlockElement
 case class GuVideoBlockElement(assets: Seq[VideoAsset], imageMedia: ImageMedia, data: Map[String, String]) extends BlockElement
 case class VideoBlockElement(data: Map[String, String]) extends BlockElement
 case class EmbedBlockElement(html: Option[String], safe: Option[Boolean], alt: Option[String]) extends BlockElement
 case class ContentAtomBlockElement(atomId: String) extends BlockElement
+case class InteractiveBlockElement(html: Option[String]) extends BlockElement
```

Comment:
It seems strange that the html properties are options here although I can see the existing models do the same. Do you know the reason here?

Reply:
Everything is probably optional because the fields are optional on the thrift models themselves

Reply:
Is it worth us coercing it into something more concrete? It doesn't feel very useful to the consumer to have a block with potentially nothing in it - although I guess filtering out the None values isn't too hard.

Reply:
The html json field won't appear if the html is None in the scala model I think. We can definitely coerse it into an empty string or something, I'd be a little concerned that that means the consumer won't have a way to differentiate between an element with blank html vs missing html if it wanted to do error handling or something on the rendering tier side but I think if the html field is blank it's a problem/error anyway.